app/controllers/games_controller.rb

```diff
@@ -9,9 +9,11 @@ def new
   end
 
   def create
-  	@game = Game.create(game_params)
+  	@game = Game.new(game_params)
+    @game.white_player = current_user
+    @game.save!
   	redirect_to game_path(@game)
-    # how do we set white_player_id to current user_id?
+   
```

Comment:
What happens if game.save fails?  Also, I'd love to see a corresponding test to back up the code.